apps/web/public/index.html

```diff
@@ -23,28 +23,21 @@
       Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
+      <script
+      src="https://uptime.betterstack.com/widgets/announcement.js"
+      data-id="144175"
+      async={true}
+      type="text/javascript"
+    />
     -->
     <title>Novu Manage Platform</title>
     <script src="%PUBLIC_URL%/env-config.js"></script>
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
-    <!--
-      This HTML file is a template.
-      If you open it directly in the browser, you will see an empty page.
-
-      You can add webfonts, meta tags, or analytics to this file.
-      The build step will place the bundled scripts into the <body> tag.
-
-      To begin the development, run `npm start` or `yarn start`.
-      To create a production bundle, use `npm run build` or `yarn build`.
-    -->
-    <script
-      src="https://uptime.betterstack.com/widgets/announcement.js"
-      data-id="144175"
-      async="async"
-      type="text/javascript"
-    ></script>
+    <% if ( process.env.REACT_APP_NOVU_MANAGED_SERVICE === 'true' ) { %>
```

Comment:
@Cliftonz I know you already merged the PR but maybe we can reuse REACT_APP_DOCKER_HOSTED_ENV here so we won't have 2 environment variables that are similar to manage.